src/pages/settings/Security/TwoFactorAuth/Steps/CodesStep.tsx

```diff
@@ -18,23 +19,31 @@ import Clipboard from '@libs/Clipboard';
 import localFileDownload from '@libs/localFileDownload';
 import StepWrapper from '@pages/settings/Security/TwoFactorAuth/StepWrapper/StepWrapper';
 import useTwoFactorAuthContext from '@pages/settings/Security/TwoFactorAuth/TwoFactorAuthContext/useTwoFactorAuth';
-import {defaultAccount, TwoFactorAuthPropTypes} from '@pages/settings/Security/TwoFactorAuth/TwoFactorAuthPropTypes';
+import type { TwoFactorAuthStepOnyxProps } from '@pages/settings/Security/TwoFactorAuth/TwoFactorAuthPropTypes';
 import * as Session from '@userActions/Session';
 import * as TwoFactorAuthActions from '@userActions/TwoFactorAuthActions';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 
-function CodesStep({account = defaultAccount, backTo}) {
+type CodesStepProps = TwoFactorAuthStepOnyxProps & {
+    /** The route to go back to when the user cancels */
+    backTo: Route | undefined;
+}
+
+function CodesStep({
+    account,
+    backTo,
+}: CodesStepProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
-    const {translate} = useLocalize();
-    const {isExtraSmallScreenWidth, isSmallScreenWidth} = useWindowDimensions();
+    const { translate } = useLocalize();
+    const { isExtraSmallScreenWidth, isSmallScreenWidth } = useWindowDimensions();
     const [error, setError] = useState('');
 
-    const {setStep} = useTwoFactorAuthContext();
+    const { setStep } = useTwoFactorAuthContext();
 
     useEffect(() => {
-        if (account.requiresTwoFactorAuth || account.recoveryCodes) {
+        if (account?.requiresTwoFactorAuth ?? account?.recoveryCodes) {
```

Comment:
I would stick to `||` operator cause `??` operator is not working when value is empty string so for example if `account.requiresTwoFactorAuth` will be `''` then this value will be taken